distortion_model: "non-fisheye"
width: 1920
height: 1080
D: [0.0796605,-0.00485538,-0.00389234,0.00280428,-0.224057]
K: [1561.59,0.0,935.159,0.0,1561.59,549.015,0.0,0.0,1]
